operations/mimir-mixin/alerts.libsonnet

```diff
@@ -9,5 +9,6 @@
     (import 'alerts/autoscaling.libsonnet') +
     (if $._config.ingest_storage_enabled then import 'alerts/ingest-storage.libsonnet' else {}) +
     (import 'alerts/continuous-test.libsonnet') +
-    (if $._config.gem_enabled then import 'alerts/gem.libsonnet' else {}),
+    (if $._config.gem_enabled then import 'alerts/gem.libsonnet' else {}) +
+    (import 'alerts/invalid-cluster-labels.libsonnet'),
```

Comment:
nit: Since this is a pretty general alert, not really specific to any component I think it's fine to go in `alerts/alerts.libsonnet`

Reply:
OK! Thanks for the feedback :)

Reply:
Done.